<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketListV</class>
 <widget class="QMainWindow" name="TicketListV">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="firstPageButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ticketInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="lastPageButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPageButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>921</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableViewItem {background: qlineargradient(spread:pad, x1:0.573909, y1:0.693, x2:0, y2:0, stop:0 rgba(195, 227, 148, 255), stop:1 rgba(255, 255, 255, 255));}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousPageButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="appliedFilters">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="filterList">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextEdit" name="filterBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: url(&quot;:/new/prefix1/loginButtonImage.png&quot;) repeat-x; 
width: 200px;  
padding:10px 0;  
color: palette(shadow); 
text-align:center; 
border:1px solid #608925;  
font-weight:bold;  
text-transform:uppercase;  
border-radius: 30px;  
color:#FFF;  
text-decoration: none;  
opacity:.85;  
}

QPushButton:hover {
border-color:#49671d;  
}


QPushButton:active {  
background-position:  0 -200px;  
padding:11px 0 9px;  
}  </string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QComboBox" name="showHideBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="numberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of Tickets:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>194</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>194</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dateTime1Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateTime2Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>to:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
